en/config_mc/racer_setup.md

```diff
@@ -135,28 +131,26 @@ As mentioned in the previous section, filters affect the control latency.
 This is the filtering pipeline for the controllers in PX4:
 - On-chip DLPF for the gyro sensor. 
   The cutoff frequency is set to 98Hz and it is sampled at 1kHz.
-- Low-pass filter on the gyro sensor data. It can be configured with the
-  [IMU_GYRO_CUTOFF](../advanced_config/parameter_reference.md#IMU_GYRO_CUTOFF) parameter.
-- Low-pass filter on the D-term. The D-term is most susceptible to noise while
-  slightly increased latency does not negatively affect performance. For this
-  reason the D-term has an additional low-pass filter, configurable via
-  [MC_DTERM_CUTOFF](../advanced_config/parameter_reference.md#MC_DTERM_CUTOFF).
-- A slewrate filter on the motor outputs ([MOT_SLEW_MAX](../advanced_config/parameter_reference.md#MOT_SLEW_MAX)). Generally not used.
-
-To reduce the control latency, we want to increase the cutoff frequency for the low-pass filters. 
-However this is a trade-off as it will also increase the noise of the signal, which is fed to the motors. 
+- Low-pass filter on the gyro sensor data. It can be configured with the [IMU_GYRO_CUTOFF](../advanced_config/parameter_reference.md#IMU_GYRO_CUTOFF) parameter.
+- Low-pass filter on the D-term. The D-term is most susceptible to noise while slightly increased latency does not negatively affect performance.
+  For this reason the D-term has an additional low-pass filter, configurable via [IMU_DGYRO_CUTOFF](../advanced_config/parameter_reference.md#IMU_DGYRO_CUTOFF).
```

Comment:
`IMU_GYRO_CUTOFF` and `IMU_DGYRO_CUTOFF` are now separately applied (i.e. the D-term does not contain `IMU_GYRO_CUTOFF`).

Reply:
@bkueng Please just make the changes that are correct for this - otherwise I will screw it up (about 28 years since I did any control or filter theory). 

Then we just need to decide if new images are needed, or the same images with the filter label changed: https://github.com/PX4/px4_user_guide/pull/703#issuecomment-606431184

Reply:
```
- A separate low-pass filter on the D-term. The D-term is most susceptible to noise while slightly increased latency does not negatively affect performance.
  For this reason the D-term has a separately configurable low-pass filter, [IMU_DGYRO_CUTOFF](../advanced_config/parameter_reference.md#IMU_DGYRO_CUTOFF).
```